visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.scss

```diff
@@ -30,26 +31,77 @@
 	}
 }
 
+.ai-suspicious-metrics-list {
+	display: flex;
+
+	.metric-button {
+		min-height: 36px;
+		line-height: 23px;
+		min-width: 45px;
+		padding: 0 12px 0 12px;
+		text-transform: none;
+		margin-right: 0;
+		overflow: visible;
+	}
+
+	.risk-button {
+		margin-left: 0;
+		min-width: 36px;
+		width: 36px;
+		height: 36px;
+		color: #9c0e6a;
+	}
+}
+
+.ai-drop-down {
+	padding: 0;
+}
+
+md-button.title {
+	color: rgba(0, 0, 0, 0.38);
+	flex-direction: column;
+}
+
+md-menu-item.title-item {
+	height: 100%;
+
+	p.title {
+		margin: 12px 0px 12px;
+		line-height: 24px;
+	}
+}
+
 md-menu-item.ai-list-item {
 	padding: 0 16px;
+	margin-bottom: 10px;
+	height: auto;
+	min-height: unset;
```

Comment:
You probably want to use min-height: auto here or initial.

https://stackoverflow.com/questions/33834049/what-is-the-difference-between-the-initial-and-unset-values